alchemy/test/cloudflare/worker.test.ts

```diff
@@ -1356,4 +1356,92 @@ describe("Worker Resource", () => {
       await destroy(scope);
     }
   });
+
+  test("deploy worker to dispatch namespace", async (scope) => {
```

Comment:
the test should also deploy a Worker that is bound to the dispatch namespace and actually calls a worker through it to make sure it works

make sure to use fetchAndExpectOK for durability